ui/react-components/components/EditAppointment/EditAppointment.jsx

```diff
@@ -203,14 +204,24 @@ const EditAppointment = props => {
 
     const isWeeklyRecurringAppointment = () => appointmentDetails.recurrenceType === weekRecurrenceType;
 
+    const setServiceErrorMessageFromResponse = response => {
+        response.error && response.error.message ? setServiceErrorMessage(response.error.message)
```

Comment:
Same as in add appointment comment.

Reply:
Can we extract to helper file, we use it in both edit and add.?

Reply:
same applies as the above

Reply:
extracting to helper is a good idea.. I will see that

Reply:
I tried but we are depending on errorTranslations.unexpectedServiceErrorMessage it has dependency on **ErrorTranslationsUtil**